docker/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM ubuntu:18.04
+MAINTAINER Haitao Liu<liuguo09@gmail.com>
+
+RUN dpkg --add-architecture i386
+RUN apt-get update
+RUN apt-get install -y git-core automake bison build-essential flex gperf wget libx11-dev libxext-dev u-boot-tools
+RUN apt-get install -y lib32z1-dev lib32ncurses5-dev libc6-dev-i386 libx11-dev:i386 libxext-dev:i386
+RUN apt-get install -y curl gettext
+RUN apt-get install -y libcurl4-openssl-dev
+RUN apt-get install -y xxd
+RUN apt-get install -y unzip
+RUN apt-get clean
+RUN mkdir /prebuilt
+
+WORKDIR /prebuilt
+
+ADD gcc-arm-none-eabi.tar.gz ./
```

Comment:
change to download from offical release website

Reply:
I'll update it later.